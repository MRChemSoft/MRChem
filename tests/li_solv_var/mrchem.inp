# vim:syntax=sh:
world_prec = 1.0e-3               # Overall relative precision
world_size = 5                    # Size of simulation box 2^n

Molecule {
  charge = 1
$coords
Li   0.0     0.0    0.0
$end
}

WaveFunction {
  method = pbe0                     # Wave function method (HF or DFT)
  restricted = true
}

Properties {
  reaction_energy = true            # Compute ground state energy
}
Environment {
  run_SCRF = true
  run_variational = true
  kain_history = 5
  Cavity {
    $coords
    0.0     0.0    0.0  4.0
    $end
    width = 0.5
    angstrom = false
  }
  Permittivity {
    epsilon_out = 2.0
    epsilon_in = 1.0
    linear = false
  }
}

SCF {
  run = true
  kain = 5                        # Length of KAIN iterative history
  orbital_thrs = 1.0e-3           # Convergence threshold in orbital residual
  guess_type = mw                 # Type of initial guess: none, mw, gto
}