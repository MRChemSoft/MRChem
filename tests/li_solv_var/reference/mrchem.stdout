                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                                                                ***
***        __  __ ____   ____ _                                    ***
***       |  \/  |  _ \ / ___| |__   ___ _ __ ___                  ***
***       | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                 ***
***       | |  | |  _ <| |___| | | |  __/ | | | | |                ***
***       |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                ***
***                                                                ***
***       VERSION            0.3.0-alpha                           ***
***                                                                ***
***       Git branch         new_solvent                           ***
***       Git commit hash    5789e0a6e118cd9a50c2                  ***
***       Git commit author  Gabriel                               ***
***       Git commit date    Tue Apr 21 17:28:00 2020 +0200        ***
***                                                                ***
***       Magnar Bjorgve     <magnar.bjorgve@uit.no>               ***
***       Roberto Di Remigio <roberto.di.remigio@uit.no>           ***
***       Luca Frediani      <luca.frediani@uit.no>                ***
***       Gabriel Gerez      <gsa017@post.uit.no>                  ***
***       Stig Rune Jensen   <stig.r.jensen@uit.no>                ***
***       Peter Wind         <peter.wind@uit.no>                   ***
***                                                                ***
**********************************************************************

----------------------------------------------------------------------

 MPI processes         :      (no bank)                             1
 OpenMP threads        :                                            8
 Total cores           :                                            8
                                                                      
----------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

----------------------------------------------------------------------

 MRCPP version         : 1.2.0-alpha3
 Git branch            : new_solvent
 Git commit hash       : 3f287c5097d6819ab1bc
 Git commit author     : Gabriel
 Git commit date       : Thu Apr 2 11:43:14 2020 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP

----------------------------------------------------------------------


                                                                      
======================================================================
                       MultiResolution Analysis
----------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
----------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                     Initializing Molecule                      ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            1
 Multiplicity          :                                            1
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***              Computing Ground State Wavefunction               ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial orbitals
 Method                : Project MW molecular orbitals
 Precision             : 1.00000e-03
 Restricted            : True
 MO file               : initial_guess/phi_p_scf
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
                          Molecular Orbitals
----------------------------------------------------------------------
 Alpha electrons       :                                            1
 Beta electrons        :                                            1
 Total electrons       :                                            2
----------------------------------------------------------------------
    n  Occ Spin        :                                         Norm
----------------------------------------------------------------------
    0    2    p        :                           1.000000000000e+00
======================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Compute initial energy
 Method                : DFT (PBE0)
 Precision             : 1.00000e-03
 Localization          : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


error:
3.471017e+00
======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.224479160090
 E-N energy            :           (au)              -16.102495490693
 Coulomb energy        :           (au)                3.286744533915
 Exchange energy       :           (au)               -0.410833798195
 X-C energy            :           (au)               -1.259291849745
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
 Tot. Reac. Energy     :           (au)               -0.082836124582
 El. Reac. Energy      :           (au)                0.165701092133
 Nuc. Reac. Energy     :           (au)               -0.248537216715
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.095696352494
 Nuclear energy        :           (au)               -0.248537216715
----------------------------------------------------------------------
 Total energy          :           (au)           -7.344233569209e+00
                       :     (kcal/mol)           -4.608576107227e+03
                       :       (kJ/mol)           -1.928228244515e+04
                       :           (eV)           -1.998467675489e+02
======================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation           : Optimize ground state orbitals
 Method                : DFT (PBE0)
 Checkpointing         : Off
 Max iterations        : 100
 KAIN solver           : 5
 Localization          : Off
 Diagonalization       : First two iterations
 Start precision       : 1.00000e-03
 Final precision       : 1.00000e-03
 Helmholtz precision   : 1.00000e-03
 Energy threshold      : Off
 Orbital threshold     : 1.00000e-03
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


======================================================================
 Iter         MO residual            Total energy              Update
----------------------------------------------------------------------
    0        1.000000e+00         -7.344233569209       -7.344234e+00
error:
7.998382e-01
error:
1.377353e-01
    1        5.814330e-03         -7.322598162628        2.163541e-02
error:
1.871925e-02
    2        4.335649e-03         -7.321803787206        7.943754e-04
error:
2.013772e-03
    3        1.058867e-03         -7.321965853701       -1.620665e-04
error:
5.202069e-05
    4        2.602901e-04         -7.321952442249        1.341145e-05
----------------------------------------------------------------------
                   SCF converged in 4 iterations!
======================================================================


                                                                      
**********************************************************************
***                                                                ***
***                 Printing Molecular Properties                  ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
======================================================================
                               Molecule
----------------------------------------------------------------------
 Charge                :                                            1
 Multiplicity          :                                            1
----------------------------------------------------------------------
    N    Atom          :              x              y              z
----------------------------------------------------------------------
    0      Li          :       0.000000       0.000000       0.000000
----------------------------------------------------------------------
 Center of mass        :       0.000000       0.000000       0.000000
 Gauge origin          :       0.000000       0.000000       0.000000
======================================================================


======================================================================
                           Orbital Energies
----------------------------------------------------------------------
    n  Occ Spin        :                                      Epsilon
----------------------------------------------------------------------
    0    2    p        :           (au)               -2.248161344326
----------------------------------------------------------------------
 Sum occupied          :           (au)               -4.496322688651
======================================================================


======================================================================
                           Molecular Energy
----------------------------------------------------------------------
 Kinetic energy        :           (au)                7.217071471489
 E-N energy            :           (au)              -16.094586524024
 Coulomb energy        :           (au)                3.285641157977
 Exchange energy       :           (au)               -0.410695934961
 X-C energy            :           (au)               -1.258832015090
 Ext. field (el)       :           (au)                0.000000000000
 Ext. field (nuc)      :           (au)                0.000000000000
 Tot. Reac. Energy     :           (au)               -0.060550597640
 El. Reac. Energy      :           (au)                0.121121703612
 Nuc. Reac. Energy     :           (au)               -0.181672301251
----------------------------------------------------------------------
 Electronic energy     :           (au)               -7.140280140997
 Nuclear energy        :           (au)               -0.181672301251
----------------------------------------------------------------------
 Total energy          :           (au)           -7.321952442249e+00
                       :     (kcal/mol)           -4.594594489079e+03
                       :       (kJ/mol)           -1.922378335478e+04
                       :           (eV)           -1.992404672238e+02
======================================================================


                                                                      
                                                                      
**********************************************************************
***                                                                ***
***                         Exiting MRChem                         ***
***                                                                ***
***                    Wall time :  0h  0m 15s                     ***
***                                                                ***
**********************************************************************
                                                                      
                                                                      
